gradio/cli/commands/hf_upload_mcp.py

```diff
@@ -0,0 +1,37 @@
+def main(url_or_space_id: str):
+    from pathlib import Path
+
+    import requests
+    from huggingface_hub import space_info
+    from mcp.server.fastmcp import FastMCP
+
+    mcp = FastMCP("hf-upload-mcp")
```

Comment:
This server needs to use stdio hence why I use fastmcp instead of gradio Interface